store/versioned_cachingstore.go

```diff
@@ -396,14 +407,6 @@ func newVersionedCachingStoreSnapshot(snapshot Snapshot, cache *versionedBigCach
 	}
 }
 
-func (c *versionedCachingStoreSnapshot) Delete(key []byte) {
```

Comment:
Why were these functions removed?

Reply:
`versionedCachingStoreSnapshot` is only used as a  `Snapshot` in GetSnapshot. 
A snapshot only has `Get, Range, Has and Release` members. So the extra versionedCachingStoreSnapshot members can't be used and only seem to confuse.